### ISSUE TITLE
Transaction options

### ISSUE THREAD

**Issue by jfedyczak** [josephg/node-foundationdb#17]
Is it possible to set transaction options? Specifically I'd like to limit transaction repeating in case of conflict.

**Comment by josephg** [comment]
Yep! Pass options as an optional second parameter to `doTransaction`

```javascript
db.doTn(async () => {...}, {retry_limit: 3})
```

**Comment by jfedyczak** [comment]
What about `NextWriteNoWriteConflictRange` option before `.set`. How to achieve sth like:

```javascript
db.doTransaction(async tn => {
    tn.setOption(NextWriteNoWriteConflictRange, true)
    tn.set(key, value)
})
```

**Comment by josephg** [comment]
```javascript
db.doTn(async () => {
  tn.setOption(fdb.TransactionOptionCode.NextWriteNoWriteConflictRange)
  tn.set('foo', 'bar')
})
```

I just published 0.6.0 to npm which exposes`fdb.TransactionOptionCode`, which wasn't public before. Give it a try

**Comment by jfedyczak** [comment]
Works fine. Thank you!
